pages/linux/systemd-analyze.md

```diff
@@ -1,6 +1,6 @@
 # systemd-analyze
 
-> Show timing details about the boot process.
+> Show timing details about the boot process(services, mount points, devices, sockets).
```

Comment:
I'd suggest actually using the word that is going to be used in the tokens, e.g. "unit". Also, this needs a space before the parenthesis :)